ui/components/Modal.tsx

```diff
@@ -14,8 +14,12 @@ type Props = {
 };
 
 const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
   background-color: ${(props) => props.theme.colors.white};
   margin: 0 auto;
+  height: 400px;
```

Comment:
Do we want ALL `Modals` to have this height?

Reply:
Yeah...the github auth and remove app modals need to be the same height. I could make a class for the two of them which again, is probably better